views/networking/private_subnet/networking.erb

```diff
@@ -66,15 +70,17 @@ viewable_fws, viewable_subnets =
               id="cps-delete-<%= subnet.ubid %>"
               class="relative whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-6"
             >
-              <%== part(
-                "components/delete_button",
-                url: "#{ps_path}/disconnect/#{subnet.ubid}",
-                confirmation_message: "Are you sure to disconnect?",
-                redirect: ps_path,
-                method: "POST",
-                text: "Disconnect"
-              ) %>
-            </td>
+              <% if show_delete %>
```

Comment:
Am I missing something or is this `show_delete` always `true`

Reply:
This was fallout from rebasing away from #3901, which did have `show_delete` not always true in this commit.